lib/index.js

```diff
@@ -77,13 +87,46 @@ export default {
     this.subscriptions.dispose();
   },
 
+  fixJob(editor) {
+    let correctScope = false;
+
+    if (editor && editor.buffer && editor.buffer.file && editor.buffer.file.path) {
+      const { path } = editor.buffer.file;
+
+      this.baseScopes.forEach((scope) => {
```

Comment:
Use logic [like this](https://github.com/AtomLinter/linter-eslint/blob/62b44afa4f5d876b6232b3dac04aba1081d8ff2a/src/validate/editor.js#L18-L21) to check if the `editor` has a cursor that is within a location that has a valid scope.